composer.json

```diff
@@ -45,7 +45,7 @@
         "drupal/editor_advanced_link": "^1.4",
         "drupal/embed": "~1.0",
         "drupal/linkit": "~5.0@beta",
-        "su-sws/stanford_media": "dev-8.x-2.x"
+        "su-sws/stanford_media": "^8.2.0"
```

Comment:
```suggestion
        "su-sws/stanford_media": "^8.2.0@alpha"
```

Reply:
they do the same.

Reply:
Sort of but this allows for alpha builds when the `minimum-stability` is set to stable instead of dev.

Reply:
cool